decentralizediso20022/src/main/resources/application-external.properties

```diff
@@ -2,13 +2,13 @@ server.port=443
 server.servlet.context-path=/api
 
 # dummy keystore
-server.ssl.key-store=src/main/resources/keystore.p12
+server.ssl.key-store=keystore.p12
 server.ssl.key-store-password=qwerty
 
 security.require-ssl=true
 
 # dummy truststore
-server.ssl.trust-store=src/main/resources/truststore.p12
+server.ssl.trust-store=truststore.p12
```

Comment:
Tomcat is responsible for TLS server support on the external component. server.ssl.key-store and server.ssl.trust-store are read by tomcat, and tomcat does not know where the resources folder is. That is why I prefixed the keystore and truststore with the path to the resources folder here, but not in the internal component.

Reply:
Because otherwise tomcat cannot find the files

Reply:
I have fixed this, but it did not solve the underlying issue.